Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -40,42 +40,48 @@ class RawPCCProducer : public edm::global::EDProducer<edm::EndLuminosityBlockPro
+          iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::string>("inputPccLabel"),
+          iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::string>("ProdInst")))),
+      lumiCorrectionsToken_(esConsumes<edm::Transition::EndLuminosityBlock>()),
+      modVeto_(
+          iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::vector<int>>("modVeto")),
+      applyCorr_(iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
+                     .getUntrackedParameter<bool>("ApplyCorrections", false)),
+      takeAverageValue_(iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
+                            .getUntrackedParameter<std::string>("OutputValue", std::string("Average"))),
+      putToken_(produces<LumiInfo, edm::Transition::EndLuminosityBlock>(
           iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
-              .getUntrackedParameter<std::string>("outputProductName", "alcaLumi"))},
-      takeAverageValue_{iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
-                            .getUntrackedParameter<std::string>("OutputValue", std::string("Totals"))},
-      modVeto_{
-          iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::vector<int>>("modVeto")},
-      csvOutLabel_{iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
-                       .getUntrackedParameter<std::string>("label", std::string("rawPCC.csv"))},
-      saveCSVFile_{iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
-                       .getUntrackedParameter<bool>("saveCSVFile", false)},
-      applyCorr_{iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
-                     .getUntrackedParameter<bool>("ApplyCorrections", false)} {
-  auto pccSource =
-      iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::string>("inputPccLabel");
-  auto prodInst =
-      iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters").getParameter<std::string>("ProdInst");
-
-  pccToken_ = consumes<reco::PixelClusterCounts, edm::InLumi>(edm::InputTag(pccSource, prodInst));
+              .getUntrackedParameter<std::string>("outputProductName", "alcaLumi"))),
+      saveCSVFile_(iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
+                       .getUntrackedParameter<bool>("saveCSVFile", false)),
+      csvOutLabel_(iConfig.getParameter<edm::ParameterSet>("RawPCCProducerParameters")
+                       .getUntrackedParameter<std::string>("label", std::string("rawPCC.csv"))) {
+  edm::LogPrint("RawPCCProducer") << " veto list size: " << modVeto_.size();
+  edm::LogPrint("RawPCCProducer") << " applyCorr: " << applyCorr_;
+  edm::LogPrint("RawPCCProducer") << " takeAverage: " << takeAverageValue_.c_str();
```

Comment:
@benitezj I think we wanted this to be LogDebug no? And then the issue was that `edm::LogDebug` doesnt exists, so in the end what you need to do is `LogDebug("RawPCCProducer")`